### ISSUE TITLE
Example for using cursor.observeChanges in a Vue component

### ISSUE THREAD

**Issue by karlitos** [meteor-vue/vue-meteor-tracker#48]
I could not find any example of how to use __cursor.observeChanges__ in a Vue component.

### Reason

I have a component encapsulating a interactive data visualization. This visualization is not reactive, the data it relies on needs to be transformed first and the layout will be computed subsequently . On the other hand it is possible to perform efficient atomic operations to modify the layout afterwards.  This is where __cursor.observeChanges__  comes in handy.

Basically I need automatic subscription with dynamic parameters which will be re-called reactively when the container component props changes. When the subscription is ready I need to create a cursor with dynamic parameter and observe the changes with the _observeChanges_ method.
